packages/config/config/devices/0x0438/1_kanal_bryter.json

```diff
@@ -2,8 +2,8 @@
 {
 	"manufacturer": "Namron AS",
 	"manufacturerId": "0x0438",
-	"label": "1 KANAL BRYTER",
-	"description": "",
+	"label": "7070990327756",
+	"description": "Namron 1 kanal bryter K2",
```

Comment:
The label should be friendlier if possible as it appears in the UIs. I'd make the Namron 1 the label and the rest the description. Please carry this through to the other files.

Also, per the style guide the description should be Title Case. Capitilize each word.